crates/criticalup-cli/tests/cli/auth_set.rs

```diff
@@ -36,9 +36,12 @@ macro_rules! run_cmd {
             .expect("failed to execute command");
         match &$expected {
             Some(expected) => {
-                // this regex replacement dance is required because this nested macro tests
+                // Since 1.84, stable Rust Clippy will complain about Regex inside a for loop.
```

Comment:
Since this is a test this is fine, but normally this would indeed be a thing worth fixing.